### PR TITLE
Updated team

### PR DESCRIPTION
**Pull request by gecofernando**
Updated team. Removed Santiago. Added Jazmin, Juan, Fabricio, German, Agustin.

### CODE REVIEW

File: source/team/agus.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+  date: 04/11/2014
```
Comment (adamantivm) [inline source/team/agus.html.erb]: why are these dates so old?

File: source/team/juan.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="person-bio">
+    <p>
+      Juan is an Electronics Engineer from the Instituto Tecnológico de Buenos Aires. When it comes 
+      to software development, he can build code for a wide variety of devices, from low-level 
+      hardware drivers for microcontrollers to applications in larger computer systems. He enjoys 
+      learning and combining new technologies when working in a project.
+    </p>
+
+    <p>
+      He spends most of his free time with his friends. He also likes playing paddle, training at
```
Comment (adamantivm) [inline source/team/juan.html.erb]: I'd call it "paddle tennis" and maybe put a link. No one outside of Argentina (or maybe Spain) will know what it is.

**Comment by gecofernando** [comment]
TBH i'm not sure what the dates were intended for. They're not shown
anywhere. The only use I found them is to organice the bios.

Good catch regarding the paddle part.

**Comment by adamantivm** [comment]
deployed

File: source/team/agus.html.erb (hunk 1 above)
Comment (basicNew) [inline source/team/agus.html.erb]: @gecofernando @adamantivm sorry for arriving late here. Given that middleman is quite oriented to blog posts (where sorting by date is the default) it is quite likely that the date is used for that (and that when the site was built that was intended as the date the person joined Ekumen maybe?). Looking at https://github.com/ekumenlabs/ekumenlabs.github.io/blob/middleman/source/team.html.erb it seems that it definitely uses some kind of ordering. 
HTH

**Pull request merged**